vendor/OWNERS

```diff
@@ -1,8 +0,0 @@
-reviewers:
```

Comment:
Why remove the OWNERS file? Was that accidental?

Reply:
!!?

Reply:
Hi @jhadvig, `vendors/OWNERS` was removed in your initial backend [commit "Add endpoint for listing operators operands + Add operator-framework/kubectl-operator package"](https://github.com/openshift/console/pull/9142/commits/b5e257850968ebe30d3db2c774dd40a49bf7f4f7#diff-c61ce090d35fbd53464a861c813260cd18686e9bd1e17b0131565b4a5aeee5d7), was this intentional?  -thanks

Reply:
![image](https://user-images.githubusercontent.com/12733153/130677811-a7cd7b07-f41f-42a8-8c92-920152a39648.png)

Reply:
hmm, not sure how that happend ?!
@dtaylor113 ould you just dd the file back? We will need to squash PR's commits anyway.

Reply:
restored